src/components/search/RatingCapsule.tsx

```diff
@@ -1,41 +1,60 @@
+import { Business } from '@futureproof/typings'
 import React from 'react'
-import { Text, View, StyleSheet } from 'react-native'
+import { StyleSheet, Text, View } from 'react-native'
 
-export const ratingValToColour = (ratingValue: number): string => {
-  if (ratingValue > 85) return '#1EA853'
-  if (ratingValue > 70) return '#50A75C'
-  if (ratingValue > 50) return '#BED62E'
-  if (ratingValue > 30) return '#D6C52E'
+export const ratingToColour = (ratingValue : number) : string => {
+  if (ratingValue < 0 || ratingValue > 100) throw new Error('ratingValue must be between 0 and 100')
```

Comment:
extract colours to external source so they can be used by other components

Reply:
and maybe rating boundaries for each colour if that is possible

Reply:
I think that this maybe outside the scope of this PR. I would recommend opening an issue though.